ddmrp_warning/views/ddmrp_warning_item_views.xml

```diff
@@ -13,7 +13,9 @@
                 decoration-danger="severity == '3_high'"
             >
                 <field name="name" />
+                <field name="product_id" />
                 <field name="severity" />
+                <field name="location_id" />
```

Comment:
can you add groups="stock.group_stock_multi_locations" to all location_id usages?

Reply:
Done!